internal/handlers/common/compare.go

```diff
@@ -122,7 +122,10 @@ func compareScalars(a, b any) compareResult {
 
 	case time.Time:
 		b, ok := b.(time.Time)
-		if ok {
+		switch {
+		case ok && a.Unix() != b.Unix():
+			return compareOrdered(a.Unix(), b.Unix())
+		case ok && a.Unix() == b.Unix():
```

Comment:
I think it can be simplified to something like:

```go
if !ok {
  return notEqual
}

return compareOrdered(a.UnixNano(), b.UnixNano())
```

To avoid `==` and `!=` there, and to simplify

Reply:
The logic has been simplified